sys-libs/timezone-data/timezone-data-2021a.ebuild

```diff
@@ -150,28 +150,29 @@ configure_tz_data() {
 	fi
 
 	if ! tz=$(get_TIMEZONE) ; then
-		einfo "Assuming your empty ${etc_lt} file is what you want; skipping update."
+		einfo "Assuming your empty ${src} file is what you want; skipping update."
 		return 0
 	fi
-	if [[ "${tz}" == "FOOKABLOIE" ]] ; then
-		elog "You do not have TIMEZONE set in ${src}."
 
-		if [[ ! -e "${etc_lt}" ]] ; then
-			cp -f "${EROOT}"/usr/share/zoneinfo/Factory "${etc_lt}"
-			elog "Setting ${etc_lt} to Factory."
-		else
-			elog "Skipping auto-update of ${etc_lt}."
-		fi
+	if [[ "${tz}" == "FOOKABLOIE" ]] ; then
+		einfo "You do not have a timezone set in ${src}; skipping update."
 		return 0
 	fi
 
-	if [[ ! -e "${EROOT}/usr/share/zoneinfo/${tz}" ]] ; then
-		elog "You have an invalid TIMEZONE setting in ${src}"
-		elog "Your ${etc_lt} has been reset to Factory; enjoy!"
-		tz="Factory"
+	local tzpath="${EROOT}/usr/share/zoneinfo/${tz}"
+
+	if [[ -f ${etc_lt} ]]; then
+		# If a regular file already exists, copy over it.
+		einfo "Copying ${tzpath} to ${etc_lt}."
+		cp -f "${tzpath}" "${etc_lt}"
+	else
+		# Otherwise, create a symlink and remove the timezone file.
+		einfo "Linking ${tzpath} at ${etc_lt}."
+		if ln -snf "${tzpath}" "${etc_lt}"; then
```

Comment:
Shouldn't this be a relative symlink? /etc/localtime -> ../usr/share/zoneinfo/foo/bar

Reply:
+1 for symlink.

Reply:
Updated.